docs/questions/query-builder/expressions/datetimediff.md

```diff
@@ -114,8 +104,6 @@ When you run a question using the [query builder](https://www.metabase.com/gloss
 If our [cheese sample data](#calculating-age) is stored in a PostgreSQL database:
 
 ```sql
-SELECT DATE_PART('month', AGE(aging_end, aging_start)) AS mature_age_months
-FROM cheese
 ```
```

Comment:
@calherries did you want me to update this example to use syntax from one of the supported DBs?

Reply:
Oops. That wasn't meant to be there, sorry! I've updated it now